packages/atomic/project.json

```diff
@@ -38,46 +15,48 @@
           "node ./scripts/process-css.mjs --config=tsconfig.lit.json ",
           "npm run-script -w=@coveo/release if-not-cdn -- \"rollup -c rollup.config.js\"",
           "esbuild src/autoloader/index.ts --format=esm --outfile=dist/atomic/autoloader/index.esm.js",
-          "esbuild src/autoloader/index.ts --format=cjs --outfile=dist/atomic/autoloader/index.cjs.js"
+          "esbuild src/autoloader/index.ts --format=cjs --outfile=dist/atomic/autoloader/index.cjs.js",
+          "rimraf ./dist/atomic/loader/package.json",
+          "tsc --noEmit --esModuleInterop --skipLibCheck ./dist/types/components.d.ts"
         ],
         "parallel": false,
-        "cwd": "packages/atomic"
+        "cwd": "{projectRoot}"
       }
     },
-    "validate:definitions": {
-      "dependsOn": ["^build", "cached:build:stencil"],
-      "inputs": [],
-      "outputs": [],
-      "executor": "nx:run-script",
+    "build-storybook": {
+      "dependsOn": ["build-stencil-lit", "cem:build", "list-assets"],
+      "executor": "nx:run-commands",
       "options": {
-        "script": "validate:definitions"
+        "commands": ["npx storybook build -o dist-storybook"],
+        "cwd": "{projectRoot}"
       }
     },
-    "release:phase1": {},
-    "release:phase3": {},
     "build": {
-      "dependsOn": [
-        "cached:build:stencil",
-        "delete-loader-package-json",
-        "validate:definitions",
-        "build-storybook"
-      ],
-      "inputs": [],
-      "outputs": [],
-      "executor": "nx:noop"
-    },
-    "create-generated-folder": {
       "executor": "nx:run-commands",
+      "outputs": [
+        "{projectRoot}/custom-elements.json",
+        "{projectRoot}/dist-storybook",
+        "{projectRoot}/dist",
+        "{projectRoot}/docs",
+        "{projectRoot}/src/components.d.ts",
+        "{projectRoot}/.stencil",
+        "{projectRoot}/src/generated"
+      ],
+      "cache": true,
       "options": {
-        "commands": ["node ./scripts/create-generated-folder.mjs"],
+        "commands": [
```

Comment:
We should use dependsOn instead, and it can just be `build-storybook` (cause `build-storybook` already depends on `build-stencil-lit` through `list-assets`).